gallery/src/components/NxTabs/NxTabsSimpleExample.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel
+} from '@sonatype/react-shared-components';
+
+const NxTableSimpleExample = () => {
+  const [activeTabId, setActiveTabId] = useState<number | null | undefined>(0);
```

Comment:
Why do the null and undefined need to be here?

Reply:
Since onTabSelect can receive `number | null | undefined`, setting the activeTabId must also allow any of those values. If you didn't want to set an invalid value you could only set the value when a number was received.